endpointmanager/cmd/CHPLpopulator/main.go

```diff
@@ -225,6 +225,21 @@ func main() {
 		log.Fatal(err)
 	}
 
+	// Save smaller copy of software products info file in the dev resources folder
+	if len(softwareInfoList) > 10 {
```

Comment:
I think we can save the whole software products info file to the dev_resources file, cause that will make it easier to test the software mapping functionality. I was more concerned about the endpoint list files having all the endpoints because i didn't want to query all of the developers endpoints just for testing (cause that has caused some issues in the past).